app/templates/components/forms/orders/order-form.hbs

```diff
@@ -136,6 +136,11 @@
                 <label for="payment_mode"><img src="/images/payment-logos/paypal.png" alt="paypal"></label>
               </div>
             {{/if}}
+            {{#if data.event.canPayByOmise}}
+              {{ui-radio name='payment_mode' label=(t 'Omise') value='Omise' current=data.paymentMode
+                         onChange=(action (mut data.paymentMode))}}
+              <label for="payment_mode"><img src="/images/payment-logos/omise.png" alt="omise"></label>
```

Comment:
If you didn't add an image, how would it appear?

Reply:
I have added an image in this PR @CosmicCoder96 So it should appear